packages/flutter_tools/lib/src/flutter_plugins.dart

```diff
@@ -1401,7 +1402,7 @@ List<PluginInterfaceResolution> resolvePlatformImplementation(
     }
   }
   if (hasResolutionError) {
-    throwToolExit('Please resolve the errors');
+    throwToolExit('Please resolve the resolution errors');
```

Comment:
I don't know that most end-developers will understand what "resolution" means in this context. How about "Please resolve the plugin implementation selection errors"?

Reply:
Sure: 272e07dbc430a5a3674c65be2b84683cac5abd0f ;D